vcompute/dell_r730xd/data/dell_r730xd.emu

```diff
@@ -12,17 +12,17 @@ sel_enable 0x20 1000 0x0a
 # Add DELL R730XD FRUs
```

Comment:
Wrong about what？
I have checked the dell_r630's emu file, and it seems like this.
Do you mean it should be 'Dell R730xd FRUs'?
@fub2